concordium-node/src/configuration.rs

```diff
@@ -44,6 +44,8 @@ pub(crate) fn is_compatible_wire_version(
 
 /// The maximum size of objects accepted from the network.
 pub const PROTOCOL_MAX_MESSAGE_SIZE: u32 = 20_971_520; // 20 MIB
+/// The maximum allowed transaction size in bytes.
+pub const MAX_TRANSACTION_MESSAGE_SIZE: u32 = 100_1000;
```

Comment:
I don't think this is 100kB as we discussed.

Also, this should not be a constant, but rather a part of this "Configuration" structure so it can be set as a runtime parameter. The default value there should be 100kB.